section2/exercises/if-statements.js

```diff
@@ -82,15 +101,26 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
+var cupsOfFlour = 3;
 var hasSauce = true;
 
 // Write your conditional here
+if (cupsOfFlour >= 2 && hasSauce) {
+  console.log("I could make a pizza.");
+} else if (cupsOfFlour < 2 || hasSauce == false) {
+  console.log("I will order a pizza instead.");
 
+}
 
 /*
-For the last exercise, an ideal solution probably uses a logical operator. 
-Did yours? Do you know what a logical operator in JavaScript is? Google it to answer
-for yourself!
+if you have BOTH 2 or more cups of flour AND sauce, you can have pizza.
+if you have less than 2 cups flour OR don't have sauce, no pizza for you.
+*/
+
+/*
+For the last exercise, an ideal solution probably uses a logical operator.
+Did yours? Do you know what a logical operator in JavaScript is? Google it to answer for yourself!
+
+** ah man, i did the thing. i confused = with == and it took a minute to figure out what the deal was.
```

Comment:
Super easy to do!

Your answer made use of the `OR` and `AND` logical operators.